src/leiningen/monolith.clj

```diff
@@ -187,15 +186,16 @@
   {:subtasks [#'info #'lint #'deps-on #'deps-of #'graph
               #'with-all #'each #'link #'unlink]}
   [project command & args]
-  (case command
-    "info"       (info project args)
-    "lint"       (lint project args)
-    "deps-on"    (deps-on project args)
-    "deps-of"    (deps-of project args)
-    "graph"      (graph project)
-    "with-all"   (with-all project args)
-    "each"       (each project args)
-    "link"       (link project args)
-    "unlink"     (unlink project)
-    (lein/abort (pr-str command) "is not a valid monolith command! Try: lein help monolith"))
+  (puget/with-options {:print-color (get-in project [:monolith :print-color] true)}
```

Comment:
I'm thinking about whether piggybacking on the puget options is the right approach here, but there are places where the plugin pretty-prints things with puget, so this is maybe the best approach if we want to control those outputs as well.